MoltenVK/MoltenVK/GPUObjects/MVKDevice.mm

```diff
@@ -1986,9 +1986,16 @@
 	_features.shaderSampledImageArrayDynamicIndexing = _metalFeatures.arrayOfTextures;
 	_features.textureCompressionBC = mvkSupportsBCTextureCompression(_mtlDevice);
 
-    if (_metalFeatures.indirectDrawing && _metalFeatures.baseVertexInstanceDrawing) {
-        _features.drawIndirectFirstInstance = true;
-    }
+	_features.drawIndirectFirstInstance = _metalFeatures.indirectDrawing && _metalFeatures.baseVertexInstanceDrawing;
+
+	mvkClear(&_vulkan12FeaturesNoExt);	// Start with everything cleared
+	_vulkan12FeaturesNoExt.samplerMirrorClampToEdge = _metalFeatures.samplerMirrorClampToEdge;
+	_vulkan12FeaturesNoExt.drawIndirectCount = false;
+	_vulkan12FeaturesNoExt.descriptorIndexing = true;
```

Comment:
It should now be possible to set this properly.

Reply:
What do you mean by this suggestion? 

`VK_EXT_descriptor_indexing` is reported as always supported, so this aligns with that.

We don't restrict descriptor indexing support to Metal arg buffer support. Are you thinking both `VK_EXT_descriptor_indexing` and `descriptorIndexing` support should be tested against something else?

Reply:
I was referring to the comment formerly on line 128, stating that this should only be set if `_metalFeatures.arrayOfTextures && _metalFeatures.arrayOfSamplers && shaderStorageBufferArrayNonUniformIndexing` are all true.

Reply:
Hmmm. Okay. I removed those tests as obsolete...but does it make sense that descriptor indexing (`VK_EXT_descriptor_indexing` and `descriptorIndexing`) only be permitted if texture, sampler, and storage buffer arrays are available after all? Would it be better to put those tests back in place?